docs/docs/examples/vector_stores/BagelAutoRetriever.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Updating Bagel Package from betabageldb to bagelML. There is nothing need to be changed in the code.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12594/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>